reactDashboard/src/components/Facts/Facts.jsx

```diff
@@ -1,39 +1,62 @@
-import React from 'react';
+import React from "react";
 import "./Facts.css";
 
 const Facts = () => {
   return (
     <div className="facts-container">
       <div className="facts-row">
         <div className="facts-component">
-          <img className="facts-icon" src="./assets/face_covering.png" alt=""/>
+          <img className="facts-icon" src="./assets/face_covering.png" alt="" />
           <div className="facts-guidance">
-            <div>Wear a <span className="highlight-facts">F</span>ace covering.</div>
+            <div>
+              Wear a <span className="highlight-facts">F</span>ace covering.
+            </div>
           </div>
         </div>
         <div className="facts-component">
-          <img className="facts-icon" src="./assets/larger_gatherings_alt.png" alt=""/>
+          <img
+            className="facts-icon"
+            src="./assets/larger_gatherings_alt.png"
+            alt=""
+          />
           <div className="facts-guidance">
-            <div><span className="highlight-facts">A</span>void crowded places.</div>
+            <div>
+              <span className="highlight-facts">A</span>void crowded places.
+            </div>
           </div>
         </div>
         <div className="facts-component">
-          <img className="facts-icon" src="./assets/washing_hands.png" alt=""/>
+          <img className="facts-icon" src="./assets/washing_hands.png" alt="" />
           <div className="facts-guidance">
-            <div><span className="highlight-facts">C</span>lean hands and surfaces regularly.</div>
+            <div>
+              <span className="highlight-facts">C</span>lean hands and surfaces
+              regularly.
+            </div>
           </div>
         </div>
         <div className="facts-component">
-          <img className="facts-icon" src="./assets/physical_distancing.png" alt=""/>
+          <img
+            className="facts-icon"
+            src="./assets/physical_distancing.png"
+            alt=""
+          />
           <div className="facts-guidance">
-            <div>Stay <span className="highlight-facts">T</span>wo metres away from other people.</div>
+            <div>
+              Stay <span className="highlight-facts">T</span>wo metres away from
+              other people.
+            </div>
           </div>
         </div>
         <div className="facts-component">
-          <img className="facts-icon" src="./assets/public_health_advice.png" alt=""/>
+          <img
+            className="facts-icon"
+            src="./assets/public_health_advice.png"
+            alt=""
+          />
           <div className="facts-guidance">
             <div>
-              <span className="highlight-facts">S</span>elf-isolate and{" "}
+              <span className="highlight-facts">S</span>elf-isolate and, if you
```

Comment:
This changes the meaning of the guidance. Perhaps 'If you have COVID-19 symptoms Self-isolate and book a test'

Reply:
That is certainly better wording than I came up with but I am trying to keep the highlighted 'S'
 on the top line, for consistency and also because it looks better.
![Screenshot 2020-10-21 at 12 19 57](https://user-images.githubusercontent.com/56888018/96712926-c1f54c00-1397-11eb-9f03-9065f13a2b10.png)